### ISSUE TITLE
Allow specifying width

### ISSUE THREAD

**Issue by ravinggenius** [dvtng/react-loading-skeleton#1]
I found after using skeletons that I nearly always want to specify a width. To get that I have to wrap each `<Skeleton />` with an extra component so I can set the width:

```jsx
<span style={{ width: '100px' }}><Skeleton /></span>
```

I made a wrapper to make this easer:

```jsx
<Span width="100px"><Skeleton /></Span>
```

However I would prefer just setting the (optional) width directly on the skeleton:

```jsx
<Skeleton width="100px" />
```

---

I'm willing to submit a pull request if you're interested in incorporating it.

**Comment by dvtng** [comment]
Can you help me understand why setting the width is necessary? Normally the skeleton is used in place of content that is already limited in its width. A key design goal is to avoid having to fiddle with the dimensions of the skeleton to avoid it getting out of sync with the dimensions of the content when it loads.

**Comment by ravinggenius** [comment]
We're using skeletons in places where the width is technically constrained (sorta), but in practice the text is pretty short. I've included some ascii art to hopefully demonstrate this. We're using a CSS grid to create this layout.

```
+-------------------------------------------------------------------+
| title                                                             |
+-------------------------------+-----------------------------------+
| something short               |                    something else |
| second line                   |                  isn't this nice? |
+-------------------------------+-----------------------------------+
| something really long, but not super long                         |
+-------------------------------------------------------------------+
```

**Comment by dvtng** [comment]
That makes sense, thanks. If you're happy to submit a PR, that would be much appreciated!

**Comment by immunity20** [comment]
How can i sumbit a PR ? i have done the width specification

**Comment by denkristoffer** [comment]
This would also make `<Skeleton />` useful inside inline elements with no set width 👍

**Comment by dvtng** [comment]
Thanks all. I've just released version 0.5.0 that adds this feature.
